### PR TITLE
refactor examples

### PR DESCRIPTION
**Pull request by shuisman**
Added more examples

### CODE REVIEW

File: example/examples_websites.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    buttons = api.get_resource(api.SCOPE_LIVE, api.PRODUCT_WEBSITES,api.RESOURCE_BUTTON)
+    first_button = buttons['items'][0]
+    print first_button['name']
+
+    # Get the feedback items for the first button of the list
+    feedback_items = api.get_resource(api.SCOPE_LIVE, api.PRODUCT_WEBSITES, api.RESOURCE_FEEDBACK, first_button['id'], iterate=True)
+    print len([item for item in feedback_items])
+
+    # ---------------------------------------
+	# Get all campaigns for this account
```
Comment (tzengerink) [inline example/examples_websites.py]: Here it's indented incorrectly.

File: example/examples_websites.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+"""Examples Usabilla for Websites"""
+
+import pprint
+import usabilla as ub
+
+if __name__ == '__main__':
+    # Create an API client with access key and secret key
+    api = ub.APIClient('YOUR-ACCESS-KEY', 'YOUR-SECRET-KEY')
+
+	# Get all buttons for this account
```
Comment (tzengerink) [inline example/examples_websites.py]: Here it's indented incorrectly.

File: example/examples_email.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    api.set_query_parameters({'limit': 1})
+
+    # Get the buttons
+    buttons = api.get_resource(api.SCOPE_LIVE, api.PRODUCT_WEBSITES, api.RESOURCE_BUTTON)
+
+    pprint.PrettyPrinter(indent=4).pprint(buttons)
+
+
+
+// Get all email widgets for this account.
```
Comment (tzengerink) [inline example/examples_email.py]: Is that JavaScript in our Python project? And that doesn't throw an exception when the example is run?

**Review by tzengerink — CHANGES_REQUESTED** [review]
Also the new created files do not have a new line at the end of the file. You can see that by the red "return" sign.

Let me know if you like me to help you out any further!

File: example/examples_apps.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+    # Get all apps forms for this account
+    forms = api.get_resource(api.SCOPE_LIVE, api.PRODUCT_APPS, api.RESOURCE_APP)
+    first_form = forms['items'][0]
+    print first_form['name']
+
+    # Get the feedback of the first app form
+    feedback = api.get_resource(api.SCOPE_LIVE, api.PRODUCT_APPS, api.RESOURCE_FEEDBACK, first_form['id'], iterate=True)
+    print len([item for item in feedback])
+
```
Comment (tzengerink) [inline example/examples_apps.py]: Still not right, not sure why it doesn't work here and it does in `example/examples_email.py`?

File: example/examples_websites.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    print len([item for item in responses])
+
+    # Get the stats of the first campaign
+    stats = api.get_resource(api.SCOPE_LIVE, api.PRODUCT_WEBSITES, api.RESOURCE_CAMPAIGN_STATS, first_campaign['id'])
+    print stats
+
+
+    # ---------------------------------------
+    # in-page is not yet available
+
```
Comment (tzengerink) [inline example/examples_websites.py]: ❔

**Comment by shuisman** [comment]
Hmm, something went seriously wrong with saving the files vs what was commit. Now everything should be the latest version.

File: example/examples_websites.py (hunk 2 above)
Comment (shuisman) [reply example/examples_websites.py]: done

File: example/examples_websites.py (hunk 1 above)
Comment (shuisman) [reply example/examples_websites.py]: done

File: example/examples_email.py (hunk 3 above)
Comment (shuisman) [reply example/examples_email.py]: done

**Review by tzengerink — APPROVED** [review]
🎉

File: example/examples_apps.py (hunk 4 above)
Comment (shuisman) [reply example/examples_apps.py]: done

File: example/examples_websites.py (hunk 5 above)
Comment (shuisman) [reply example/examples_websites.py]: done

**Pull request merged**